installer/templates/new/widgets/text/text.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import Widget from '../../assets/javascripts/widget';
+import {updatedAt} from '../../assets/javascripts/helpers';
 
 import './text.scss';
 
 Widget.mount(class Text extends Widget {
+  status() {
+    if (!this.state.status) { return ""; }    
+    return`status-${this.state.status}`;
+  }
   render() {
     return (
-      <div className={this.props.className}>
-        <h1 className="title">{this.props.title}</h1>
+      <div className={`${this.props.className} status-${status}`}>    
```

Comment:
so now you would pass from the job a non-class-coupled string to trigger the class in the widget:

```
status: "danger" // or "warning"
```